debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -236,6 +236,17 @@
 #djDebug .djDebugPanelContent .djdt-loader {
     display: block;
```

Comment:
Now that this is `<div>`, can remove display block as it is the default.

Reply:
Good point, I removed it.